docs/AUTHORS.md

```diff
@@ -0,0 +1,17 @@
+[comment]: <> (Don't edit this file!)
+[comment]: <> (It is automatically updated after every release of https://github.com/47degrees/.github)
+[comment]: <> (If you want to suggest a change, please open a PR or issue in that repository)
+
+# Authors
+
+## Maintainers
+
+The maintainers of the project are:
+
+@COLLABORATORS@
```

Comment:
@juanpedromoreno All the files in the `docs/` directory are full of placeholders. Is that expected?

Reply:
Yes, they are going to be used by other workflows.